File: app/src/main/java/example/com/crackle/model/Image.java
package example.com.crackle.model;

import com.google.gson.annotations.SerializedName;

public class Image {

    @SerializedName("file_path")
    private String filePath;

    public Image(String filePath) {
        this.filePath = filePath;
    }

    public String getFilePath() {
        return filePath;
    }
}

File: app/src/main/java/example/com/crackle/model/Image.kt
package example.com.crackle.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Image(@field:SerializedName("file_path")
            val filePath: String) : Parcelable
